src/utils/css.test.ts

```diff
@@ -37,6 +37,24 @@ describe('when loading the css', () => {
       expect(link.rel).toEqual('stylesheet');
     });
   });
+  describe('with "next"', () => {
+    const url = 'https://js.arcgis.com/next/esri/css/main.css';
```

Comment:
After merging #187 I don't see any conflicts, but I would expect that this now needs to be `https://js.arcgis.com/next/esri/themes/light/main.css`